platform-includes/logs/usage/dotnet.mdx

```diff
@@ -0,0 +1,61 @@
+Once the feature is enabled on the SDK and the SDK is initialized, you can send logs using the `SentrySdk.Experimental.Logger` APIs.
+
+The `SentrySdk.Experimental.Logger` instance exposes six methods that you can use to log messages at different log levels: `Trace`, `Debug`, `Info`, `Warning`, `Error`, and `Fatal`.
```

Comment:
When using MEL, people would likely be using the MEL log levels - so we should describe these here instead.

Reply:
chat: we will exclude the `SentrySdk.Experimental.Logger` APIs for Platforms that support `ILogger<T>`, to keep it concise